src/main/java/adris/altoclef/butler/ButlerConfig.java

```diff
@@ -51,4 +51,19 @@ public static ButlerConfig getInstance() {
      */
     public boolean whisperFormatDebug = false;
 
+    /**
+     * If set to true, will reply with "Sorry, you're not authorized!".
+     *
+     * Disable this if you need to stay undercover.
+     */
+    public boolean sendAuthorizationResponse = true;
+
+    /**
+     * Use this to customize the response if "sendAuthorizationResponse"
+     * 
+     * You may use {from} to get sender username.
+     * 
+     * Disable this if you need to stay undercover.
+     */
+    public String authorizationResponse = "Sorry {from} but you are not authorized!";
```

Comment:
I would replace this variable name with "failedAuthorizationResposne" that way we know this is specific to the failed condition.